doc/examples/plot_selective_search.py

```diff
@@ -0,0 +1,151 @@
+"""
+================
+Selective Search
```

Comment:
In this example I use the rag_mean_color() and the callbacks from the RAG Merging example in the gallery. I did this thinking familiarity with the other RAG examples in the gallery and to keep the example code short. Instead, would it be better to use region_adjacency_graph(), so will need another callback? Similar to the original paper on Selective Search (referenced in the example) should I use other similarity measures, say based on colour, texture, fill and size? These will make the example longer, but potentially more useful to others, and will also need some supporting functions like histogram_intersection to calculate the weights. Thoughts?